Makefile

```diff
@@ -40,3 +42,14 @@ endif
 
 test: layer/bcdhive_generated.go format lint
 	$(GOCMD) test -parallel=1 -count=1 -v ./...
+
+# Ensure workdir is clean and build image from .git
+docker-build-source-image-windows:
```

Comment:
Unrelated?

Reply:
Oops, totally forgot to to cover this. I'll separate this out into a different PR.

Reply:
Reverted for now, I'll bring it back separately.